### ISSUE TITLE
The code

### ISSUE THREAD

**Issue by zachariahyoung** [OlegDokuka/refactor-to-reactive#1]
When do you plan on adding the code? Your video is pretty sweet and I would love to see the code.

**Comment by OlegDokuka** [comment]
@zachariahyoung Hey, buddy, take a look here - > https://github.com/CollaborationInEncapsulation/get-reactive-with-spring5-demo/tree/migration-to-reactive?files=1

I will move some part of the code to here soon. But for now, all mentioned code is in ☝️

**Comment by OlegDokuka** [comment]
Also, there is commit's history with which I played during the video https://github.com/CollaborationInEncapsulation/get-reactive-with-spring5-demo/commits/migration-to-reactive

**Comment by OlegDokuka** [comment]
@zachariahyoung Let me know whether it is something you were looking for

**Comment by zachariahyoung** [comment]
Yes this is what I was looking for.  Still reviewing it.  How is your book coming?  Let me know if you need helping review it.

**Comment by OlegDokuka** [comment]
Hey buddy, can we continue our conversation using messenger or via email?

**Comment by zachariahyoung** [comment]
What messenger tool do you prefer?

hangout
facebook
twitter

Zach

**Comment by OlegDokuka** [comment]
facebook and twitter, please DM me on twitter (https://twitter.com/OlehDokuka) and we will continue that discussion here.
